packages/web-runtime/src/mixins/navigationMixin.js

```diff
@@ -14,7 +14,7 @@ export default {
      * @param permittedMenus
      * @returns {*}
      */
-    navigation_getMenuItems(permittedMenus) {
+    navigation_getMenuItems(permittedMenus, activeRoute) {
```

Comment:
```suggestion
    navigation_getMenuItems(permittedMenus, activeRoutePath) {
```
otherwise I'd expect an object, not a path.